### DISCUSSION TITLE
Welcome to ha-aquanta Discussions!

### DISCUSSION THREAD

**Discussion by ReplicaObscura** [bmcclure/ha-aquanta#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by homeautomator2** [comment]
Thanks for this integration!  I just left Homeseer, partly because the dev community is so much more active here.  I have two Aquanta's at a vacation rental.  Aquanta has been talking about "working on the API" for YEARS.  Thank you so much for reverse-engineering this.  Awesome!
